src/Jackett.Common/Definitions/matrix.yml

```diff
@@ -0,0 +1,175 @@
+---
+id: matrix
+name: Matrix
+description: "M.A.T.R.I.X is a HUNGARIAN Private Tracker for MOVIES / TV / GENERAL"
+language: hu-HU
+type: private
+encoding: UTF-8
+links:
+  - https://matrixworld.info/
+
+caps:
+  categorymappings:
+    - {id: 23, cat: Movies/Other, desc: "Animációk"}
+    - {id: 61, cat: Movies/Other, desc: "Cam/EN"}
+    - {id: 60, cat: Movies/Other, desc: "Cam/HU"}
+    - {id: 45, cat: Books/EBook, desc: "eBook/EN"}
+    - {id: 40, cat: Books/EBook, desc: "eBook/HU"}
+    - {id: 20, cat: Movies/DVD, desc: "Film/DVD9"}
+    - {id: 48, cat: Movies/HD, desc: "Film/EN/1080p"}
+    - {id: 32, cat: Movies/HD, desc: "Film/EN/720p"}
+    - {id: 38, cat: Movies/BluRay, desc: "Film/EN/Blu-ray"}
+    - {id: 27, cat: Movies/DVD, desc: "Film/EN/DVD-R"}
+    - {id: 24, cat: Movies/SD, desc: "Film/EN/SD"}
+    - {id: 34, cat: Movies/HD, desc: "Film/HU/1080p"}
+    - {id: 31, cat: Movies/HD, desc: "Film/HU/720p"}
+    - {id: 35, cat: Movies/BluRay, desc: "Film/HU/Blu-ray"}
+    - {id: 26, cat: Movies/DVD, desc: "Film/HU/DVD-R"}
+    - {id: 5, cat: Movies/SD, desc: "Film/HU/SD"}
+    - {id: 4, cat: PC/Games, desc: "Játékok/ISO"}
+    - {id: 39, cat: PC/Games, desc: "Játékok/Rip/Dox"}
+    - {id: 47, cat: Other, desc: "Klippek"}
+    - {id: 30, cat: Audio/Lossless, desc: "Lossless/EN"}
+    - {id: 29, cat: Audio/Lossless, desc: "Lossless/HU"}
+    - {id: 25, cat: Audio/MP3, desc: "MP3/EN"}
+    - {id: 6, cat: Audio/MP3, desc: "MP3/HU"}
+    - {id: 33, cat: PC, desc: "Program/egyéb"}
+    - {id: 1, cat: PC/ISO, desc: "Program/ISO"}
+    - {id: 36, cat: TV/HD, desc: "Sorozat/EN/HD"}
+    - {id: 49, cat: TV/SD, desc: "Sorozat/EN/SD"}
+    - {id: 28, cat: TV/HD, desc: "Sorozat/HU/HD"}
+    - {id: 7, cat: TV/SD, desc: "Sorozat/HU/SD"}
+    - {id: 9, cat: XXX, desc: "XXX"}
+    - {id: 44, cat: XXX/Other, desc: "XXX/HD"}
```

Comment:
```suggestion
    - {id: 44, cat: XXX/x264, desc: "XXX/HD"}
```

Reply:
I don't understand this one, can you elaborate? I wouldn't think `HD` is synonymous with `x264`, plenty of SD `x264` encodes out there. Did I miss a clue in the HTML?

Reply:
As there isn't an `HD` cat, so we generally use `x264` instead - https://github.com/search?q=repo%3AJackett%2FJackett+%22xxx%2Fx264%22+language%3AYAML&type=code&l=YAML

It's a bit of a layover from when HD encodes used x264 in an MP4 container, while SD stuck with DivX in AVI. Times have changed, but we still don't have an `HD` cat, so 🤷

Reply:
K, I mean really I couldn't bother to care, but I appreciate the education!